Although the following patch has been addressed in Icehouse 2014.1, it
still has not yet been released for Havana.  It has been modified to
apply cleanly into our current Havana implementation

commit 5c321d758c9718d7dde555316ac4fbd2f7acf424
Author: Dan Prince <dprince@redhat.com>
Date:   Mon Oct 7 12:41:28 2013 -0400

    Drop conf_key_mgr warning message!
    
    By default ConfKeyManager logs tons of WARNING message stating
    that it isn't production ready...
    
    Given that it is currently the only Cinder key manager option
    which can be used/selected I don't think repeatedly logging
    warnings is helpful. Lets just drop the warning message
    for now and when a good "production ready" cinder key manager
    implementation is implemented perhaps we can re-add a warning to
    this class (hopefully making the production ready impl the default).
    
    Change-Id: Id1fdddc20a963f9fa4749ad57f355cd83d0e14e3
    Closes-Bug: #1236459

diff --git a/cinder/keymgr/conf_key_mgr.py b/cinder/keymgr/conf_key_mgr.py
index 7b53e0c..1c9ff2d 100644
--- a/cinder/keymgr/conf_key_mgr.py
+++ b/cinder/keymgr/conf_key_mgr.py
@@ -64,8 +64,6 @@ class ConfKeyManager(key_mgr.KeyManager):
     """
 
     def __init__(self):
-        LOG.warn(_('This key manager is insecure and is not recommended for '
-                   'production deployments'))
         super(ConfKeyManager, self).__init__()
 
         self.key_id = '00000000-0000-0000-0000-000000000000'
